common/big_messaging/src/test/scala/uk/ac/wellcome/bigmessaging/fixtures/BigMessagingFixture.scala

```diff
@@ -52,9 +62,27 @@ trait BigMessagingFixture
     testWith(stream)
   }
 
+  /** Send a MessageNotification to SQS.
+    *
+    * As if another application had used a MessageWriter to send the message
+    * to an SNS topic, which was forwarded to the queue.  We don't use a
+    * MessageWriter instance because that sends to SNS, not SQS.
+    *
+    * We always send an InlineNotification regardless of size, which makes for
+    * slightly easier debugging if queue messages ever fail.
+    *
+    */
+  def sendMessage[T](queue: Queue, obj: T)(
+    implicit encoder: Encoder[T]): SendMessageResult =
+    sendNotificationToSQS[MessageNotification](
+      queue = queue,
+      message = InlineNotification(jsonString = toJson(obj).get)
+    )
+
   def withSqsBigMessageSender[T, R](bucket: Bucket,
                                     topic: Topic,
-                                    senderSnsClient: AmazonSNS = snsClient)(
+                                    senderSnsClient: AmazonSNS = snsClient,
+                                    store: Option[MemoryTypedStore[ObjectLocation, T]] = None)(
```

Comment:
Not a biggy, but you could just pass the `MemoryTypedStoreCompanion[ObjectLocation, T]()` as a default here rather than wrapping in an `Option[_]`.

(I only found out that the RHS gets evaluated each call when reviewing [this](https://github.com/wellcometrust/storage-service/pull/282#discussion_r308167094))